catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/UpdateRequestImpl.java

```diff
@@ -92,6 +92,23 @@ public UpdateRequestImpl(String[] ids, List<Metacard> metacards)
     }
 
     /**
+     * Instantiates a new UpdateRequestImpl from a set of destinations,
+     * an array of ids, and a list of {@link Metacard}
+     *
+     * @param destinations the destination ids this request should be sent to
+     * @param ids          the ids associated with the {@link Metacard} list
```

Comment:
This feels like an implementation detail that shouldn't be exposed outside this class, but since we already have other constructors doing the same thing...

❗️ Should we change the `formatEntryList()` method to validate that the list of metacard IDs or URLs match with the metacards provided (`ids[i] == metacards.get(i).getId()`) to make sure every ID or URL maps to the correct metacard? Otherwise we may end up an ID or URL that maps to the wrong metacard and that seems pretty bad to me. @pklinef, @coyotesqrl, thoughts?